lib/compress/zstd_ldm.c

```diff
@@ -379,9 +376,11 @@ static size_t ZSTD_ldm_generateSequences_internal(
 
         /* No match found -- continue searching */
         if (bestEntry == NULL) {
-            ZSTD_ldm_makeEntryAndInsertByTag(ldmState, rollingHash,
-                                             hBits, curr,
-                                             *params);
+            ldmEntry_t entry;
+
+            entry.offset = curr;
+            entry.checksum = checksum;
+            ZSTD_ldm_insertEntry(ldmState, hash, entry, *params);
```

Comment:
We also insert when we find an entry right? Could we use `ZSTD_ldm_insertEntry()` in that case too?

Reply:
Yes, I thought it'd not bring much of a win but for the sake of consistency I updated it.